--- file1.py	2021-04-28 18:51:16.238283744 +0530
+++ file3.py	2021-04-28 18:54:27.525298897 +0530
@@ -2,6 +2,7 @@
 
 print("We will see usage of diff in unix")
 print("The diff function takes the following syntax")
-print("diff file1.py file2.py .This command will print out the changes between the two files")
+#print("diff file1.py file2.py .This command will print out the changes between the two files")
 print("The above lines existed earlier")
 print("=================================")
+print("In file3 I added this line and commented a line from above")
